test/integration/editor.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { test, expect, Page, Locator } from '@playwright/test';
+import createApp from '../utils/createApp';
+import generateId from '../utils/generateId';
+import { canvasFrame, componentCatalog, pageRoot, pageOverlay } from '../utils/locators';
+import twoFieldsDomTemplate from '../domTemplates/twoTextFields.json';
```

Comment:
I prefer not to organize this using a big `/domTemplates/...` folder containing doms for all the tests, that doesn't scale well. The json should just live right next to the test. Can we rather do something like the following?

```
test
  integration
    appRename
      index.spec.ts
    editor
      index.spec.ts
      input.json
    smoke
      index.spec.ts
```

Reply:
sure, that sounds fine too